### ISSUE TITLE
Rename "Storybook" repo to publisher-storybook or similar

### ISSUE THREAD

**Issue by diversemix** [libero/publisher#309]
As a reviewer developer because we also use storybook we'd like to reduce any confusion that could arise with the name of this repository.

**Comment by diversemix** [comment]
@davidcmoulton / @discodavey  ... Just thought I'd raise the point that reading through the list of repos this one is ambiguous... let me know your thoughts

**Comment by discodavey** [comment]
I agree.  If we have the naming convention like you said of publisher-storybook or reviewer-storybook this will make it clearer.

**Comment by thewilkybarkid** [comment]
This applies pretty much _everywhere_...

**Comment by giorgiosironi** [comment]
Meeting set to choose a strategy across teams for this.
